accounts/api/serializers.py

```diff
@@ -1,8 +1,45 @@
 from django.contrib.auth.models import User
-from rest_framework import serializers
-
+from rest_framework import serializers, exceptions
 
 class UserSerializer(serializers.ModelSerializer):
     class Meta:
         model = User
-        fields = ('id', 'username', 'email')
\ No newline at end of file
+        fields = ('id', 'username', 'email')
+
+class LoginSerializer(serializers.Serializer):
+    username = serializers.CharField()
+    password = serializers.CharField()
+
```

Comment:
Please add validate function also in the login serializer, i.e. check if the user name exists or not